app/assets/stylesheets/modules/_course.styl

```diff
@@ -210,4 +210,17 @@ rating(color)
   display: none;
 
 .multi-wiki input[readonly]
-  display: none
\ No newline at end of file
+  display: none
+
+.modal-course-details
+    .pop__container
+        form
+          display flex
+          align-items center
+          justify-content center
+        .pop.open
```

Comment:
Why is this styling only in the `open` mode?

It looks like this is causing a problem with the transition animation: the modal shifts out of column format for a brief flash as the popover closes.